src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -448,6 +448,47 @@ def setUp(self):
         self.d_log = DaosLog(self.context)
         self.test_log.daos_log = self.d_log
 
+        # Add test binaries and daos binaries to PATH
```

Comment:
@mallove79  is this code specific to the cart env?  If it is should it be included in the cart_utilities and not here?

Reply:
Hi @mjean308  -- Except for a few `dfuse` `ioil` tests, I see only cart tests in `/usr/lib/daos/TESTING/tests`.  So though the name of the directory is generic (`daos/TESTING/tests`), it does seem to be mostly a directory for cart tests. Is it possible more non-CaRT tests could wind up in the "tests" dir?  In that case, this `PATH`-appending code would no longer belong in `CartTest`.

Reply:
https://github.com/daos-stack/daos/pull/4665/commits/13a66e83efcc7ce2b4c21a420ed2dfc7c65a3269